app/src/main/java/org/p2p/wallet/sell/ui/lock/SellTransactionViewDetails.kt

```diff
@@ -14,6 +14,7 @@ data class SellTransactionViewDetails(
     val formattedFiatAmount: String,
     val fiatAbbreviation: String,
     val receiverAddress: String,
+    val updatedAt: String? = null,
```

Comment:
it's better to have all fields non-nullable in domain object
and i don't think it can be null from the server

Reply:
its model for SellTransactionDetailsBottomSheet. updatedAt nullable because on SellLockedFragment not need show date subtitle